File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/broker/BrokerAccountApi.java
package com.bitget.openapi.api.broker;

import com.bitget.openapi.dto.request.broker.*;
import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Query;

public interface BrokerAccountApi {

    /**
     * get broker info
     * @return ResponseResult
     */
    @GET("/api/broker/v1/account/info")
    Call<ResponseResult> info();

    /**
     * sub create
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-create")
    Call<ResponseResult> subCreate(@Body BrokerSubCreateReq req);


    /**
     * get sub info list
     * @return ResponseResult
     */
    @GET("/api/broker/v1/account/sub-list")
    Call<ResponseResult> subList(@Query("pageSize")int pageSize,
                                 @Query("lastEndId") String lastEndId,
                                 @Query("status")String status);


    /**
     * sub modify
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-modify")
    Call<ResponseResult> subModify(@Body BrokerSubModifyReq req);


    /**
     * sub bind email
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-modify-email")
    Call<ResponseResult> subModifyEmail(@Body BrokerSubModifyEmailReq req);


    /**
     * get sub email
     * @return ResponseResult
     */
    @GET("/api/broker/v1/account/sub-email")
    Call<ResponseResult> subEmail(@Query("subUid") String subUid);

    /**
     * get sub spot account list
     * @return ResponseResult
     */
    @GET("/api/broker/v1/account/sub-spot-assets")
    Call<ResponseResult> subSpotAssets(@Query("subUid") String subUid);


    /**
     * get sub future account list
     * @return ResponseResult
     */
    @GET("/api/broker/v1/account/sub-future-assets")
    Call<ResponseResult> subFutureAssets(@Query("subUid") String subUid,
                                         @Query("productType") String productType);


    /**
     * get sub deposit address
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-address")
    Call<ResponseResult> subDepositAddress(@Body BrokerSubAddressReq req);

    /**
     * sub withdraw
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-withdrawal")
    Call<ResponseResult> subWithdraw(@Body BrokerSubWithdrawReq req);


    /**
     * sub config auto transfer
     * @return ResponseResult
     */
    @POST("/api/broker/v1/account/sub-auto-transfer")
    Call<ResponseResult> subAutoTransfer(@Body BrokerSubTransferReq req);



}

File: bitget-php-sdk-api/src/api/broker/BrokerAccountApi.php
<?php

namespace bitget\api\broker;

use bitget\model\broker\BrokerCreateSubReq;
use bitget\model\broker\BrokerModifySubReq;
use bitget\model\broker\BrokerModifyEmailReq;
use bitget\model\broker\BrokerSubDepositAddressReq;
use bitget\model\broker\BrokerSubWithdrawReq;
use bitget\model\broker\BrokerSubTransferReq;


class BrokerAccountApi
{
    const BASE_URL = "/api/broker/v1/account";

    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }


    /**
     * Get broker account  information
     * @return string
     */
    public function account(): string
    {
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/info");
    }


    /**
     * broker create sub
     * @param $brokerCreateSubReq
     * @return string
     */
    public function createSub(BrokerCreateSubReq $brokerCreateSubReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-create", $brokerCreateSubReq);
    }


    /**
     * Get sub list
     * @param  $symbol
     * @param $marginCoin
     * @return string
     */
    public function getSubList(string $pageSize, string $lastEndId, string $status): string
    {
        $params = array("pageSize" => $pageSize, "lastEndId" => $lastEndId, "status"=>$status);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/sub-list", $params);
    }

    /**
     * broker modify sub
     * @param $brokerModifySubReq
     * @return string
     */
    public function subModify(BrokerModifySubReq $brokerModifySubReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-modify", $brokerModifySubReq);
    }

    /**
     * broker modify sub
     * @param $brokerModifyEmailReq
     * @return string
     */
    public function subModifyEmail(BrokerModifyEmailReq $brokerModifyEmailReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-modify-email", $brokerModifyEmailReq);
    }

    /**
     * Get sub email
     * @return string
     */
    public function getSubEmail(string $subUid): string
    {
        $params = array("subUid" => $subUid);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/sub-email", $params);
    }


    /**
     * Get sub spot assets list
     * @return string
     */
    public function getSubSpotAssets(string $subUid): string
    {
        $params = array("subUid" => $subUid);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/sub-spot-assets", $params);
    }

    /**
     * Get sub future assets list
     * @return string
     */
    public function getSubFutureAssets(string $subUid): string
    {
        $params = array("subUid" => $subUid);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/sub-future-assets", $params);
    }


    /**
     * broker get sub deposit address
     * @param $brokerModifyEmailReq
     * @return string
     */
    public function subDepositAddress(BrokerSubDepositAddressReq $brokerSubDepositAddressReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-address", $brokerSubDepositAddressReq);
    }

    /**
     * broker sub withdraw
     * @param $brokerModifyEmailReq
     * @return string
     */
    public function subWithdraw(BrokerSubWithdrawReq $brokerSubWithdrawReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-withdrawal", $brokerSubWithdrawReq);
    }

    /**
     * broker auto transfer
     * @param $brokerModifyEmailReq
     * @return string
     */
    public function subAutoTransfer(BrokerSubTransferReq $brokerSubTransferReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/sub-auto-transfer", $brokerSubTransferReq);
    }

}